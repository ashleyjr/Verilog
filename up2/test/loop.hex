64646464444544646464644C4E4C64646464011792EA240A1F824A2E