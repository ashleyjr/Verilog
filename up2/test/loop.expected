0 0
1 0
2 0
3 0
4 0
5 0
6 0
7 0
8 0
9 0
A 0
B 0
C 0
D 0
E 0
F 0
